mysql-test/suite/rpl/t/rpl_bug72457_innodb-master.opt

```diff
@@ -1,2 +1,3 @@
 --internal-tmp-disk-storage-engine=InnoDB
 --default-tmp-storage-engine=InnoDB
+--log-error=$MYSQLTEST_VARDIR/tmp/rpl_bug72457.err
```

Comment:
This is still not good enough in the case of MTR --repeat: the testcase should restart the server in the .test file itself with the custom path, then restart in the end with default, remove the custom error log